homework_2/task1.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+int recFib(int num)
+{
+    if (num <= 1)
+    {
+        return 1;
+    }
+    return recFib(num - 2 ) + recFib(num - 1);
+}
+int iterFib(int num)
+{
+    int last = 1;
+    int first = 1;
```

Comment:
```suggestion
    int current = 1;
    int previous = 1;
```
Странный выбор имён. Разве `first` является первым элементом?